types/brotli/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for brotli 1.3
+// Project: https://github.com/foliojs/brotli.js
+// Definitions by: Artur Androsovych <https://github.com/arturovt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
```suggestion
```
the 3.2 is now default, min version, I'm not sure what we could gain here by requesting older version (nothing specific seems to be used)